core/src/main/scala/chisel3/ModuleImpl.scala

```diff
@@ -908,6 +931,18 @@ package experimental {
           case None    => getRef.name
           case Some(c) => getRef.fullName(c)
         }
+  }
+}
 
+object withModulePrefix {
```

Comment:
Is this the public entry point? Scaladoc?

should this be called `withNestedModulePrefix`? Is there a `stripPrefix`?

Reply:
Agreed that this should have ScalaDoc, I am happy with the API (no need to specify that it's `nested`, that's just the API, and we can add `stripPrefix` later if needed).